screens/OffersAndRequestsScreen.js

```diff
@@ -9,14 +9,12 @@ import {
   View
 } from "react-native";
 import Colors from "../constants/Colors";
-import { useNavigation } from "@react-navigation/native";
 import { ScrollView } from "react-native-gesture-handler";
-
 import { gql } from "apollo-boost";
 import { useQuery } from "@apollo/react-hooks";
 
 export default function OffersAndRequestsScreen(props) {
-  const navigation = useNavigation();
```

Comment:
Doing this helps testing significantly!

Reply:
All screens automatically have navigation so you don't need to import it. They are still using navigation, though!